styles.css

```diff
@@ -1 +1,57 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.grid-container {
+    background-color: antiquewhite;
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto repeat(5, 50px);
+    grid-template-areas:"a" "b" "c" "d" "e" "empty" "f";
+}
+
+@media screen and (min-width: 576px) {
+    .grid-container {
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 50px 190px 100px auto 50px;
+        grid-template-areas:
+                        "a c c"
+                        "b c c"
+                        "b e d"
+                        "b e empty"
```

Comment:
(regel 9 en 20): om een leeg plek aan te duiden in je grid, gebruik je een `.` in plaats van `empty`.